libraries/GCS_MAVLink/GCS.h

```diff
@@ -308,6 +308,13 @@ class GCS_MAVLINK
     void send_gimbal_report() const;
     void send_home_position() const;
     void send_gps_global_origin() const;
+
+    struct Attitude_Target_Info{
```

Comment:
Should this be `ModeGuided::TargetAttitude`?

'though considering things like the 6DOF frame, the data being conveyed isn't actually complete now-adays.